applications/RANSApplication/python_scripts/test_utilities.py

```diff
@@ -0,0 +1,62 @@
+import KratosMultiphysics as km
+import KratosMultiphysics.KratosUnittest as UnitTest
+from KratosMultiphysics.RANSApplication.rans_analysis import RANSAnalysis
+
+import os
+
+def RunParametericTestCase(settings_file_name,
+                           work_folder,
+                           parameters_dict,
+                           print_output=False):
```

Comment:
```suggestion
def RunParametricTestCase(
    settings_file_name,
    work_folder,
    parameters_dict,
    print_output=False):
```

Reply:
Agreed, and changed locally. Will push once you finish review :)

Reply:
done :)